integration_tests/cairo_files/test.small.cairo

```diff
@@ -1,6 +1,44 @@
-%builtins range_check
+%builtins range_check bitwise
```

Comment:
how can use use layout small for this file ? Bitwise is not in the small layout right?

Reply:
you can't. PR is in draft :p